DHFPROD-10289: Incorrect position of the information modal

amejiaq opened the pull request:
### Description

#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran newly added/edited cypress tests on Firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [n/a] Added to Release Wiki

briantang reviewed (changes requested):
Looks like your solution also removed the keyboard navigation selection (SPACE/ENTER to open/close the info modal). Can that be fixed in this same PR?

amejiaq commented:
> Looks like your solution also removed the keyboard navigation selection (SPACE/ENTER to open/close the info modal). Can that be fixed in this same PR?

Fixed 🛠🛠🛠
